api_tests/users/views/test_user_settings.py

```diff
@@ -137,3 +137,135 @@ def test_exceed_throttle(self, mock_mail, app, user_one, url, payload):
 
         res = app.post_json_api(url, payload, auth=user_one.auth, expect_errors=True)
         assert res.status_code == 429
+
+
+@pytest.mark.django_db
+class TestUserChangePassword:
+
+    @pytest.fixture()
+    def user_one(self):
+        user = AuthUserFactory()
+        user.set_password('password1')
+        user.auth = (user.username, 'password1')
+        user.save()
+        return user
+
+    @pytest.fixture()
+    def url(self, user_one):
+        return '/{}users/{}/settings/password/'.format(API_BASE, user_one._id)
+
+    @pytest.fixture()
+    def payload(self, user_one):
+        return {
+            'data': {
+                'type': 'user_password',
+                'id': '{}'.format(user_one._id),
+                'attributes': {
+                    'existing_password': 'password1',
+                    'new_password': 'password2',
+                }
+            }
+        }
+
+    def test_get(self, app, user_one, url):
+        res = app.get(url, auth=user_one.auth, expect_errors=True)
+        assert res.status_code == 405
+
+    def test_post(self, app, user_one, user_two, url, payload):
+        # Logged out
+        res = app.post_json_api(url, payload, expect_errors=True)
+        assert res.status_code == 401
+
+        # Logged in, requesting export for another user
+        res = app.post_json_api(url, payload, auth=user_two.auth, expect_errors=True)
+        assert res.status_code == 403
+
+        # Logged in
+        res = app.post_json_api(url, payload, auth=user_one.auth)
+        assert res.status_code == 204
+        user_one.reload()
+        assert user_one.check_password('password2')
+
+    def test_post_validation_old_password_invalid(self, app, user_one, url, payload):
```

Comment:
`test_post_validation_old_password_invalid`, `test_post_validation_not_all_the_same`, `test_post_validation_not_email`, `test_post_validation_not_blank`, `test_post_validation_not_too_short`, `test_post_validation_not_too_long` are all testing the behavior of `OSFUser.change_password` rather than the behavior of the view. Can you make these into more focused unit tests and remove these integration tests? 

Glancing at `osf_tests/test_user:ChangeTestPassword`, it looks like much of this functionality is already tested. So you can just add the missing pieces there.